src/main/scala/nl/knaw/dans/easy/fedoratobag/FileInfo.scala

```diff
@@ -57,8 +64,8 @@ object FileInfo {
 
       new FileInfo(
         foXml \@ "PID",
-        Paths.get(get("path")),
-        get("name"),
+        Paths.get(replaceNonAllowedCharacters(get("path"))),
+        replaceNonAllowedCharacters(get("name")),
```

Comment:
Please add/modify unit tests, or describe how you tested this manually on deasy. See where this mockup is used https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/4753ddb6c64cbab98505ffeb6fc997f403f34ba2/src/test/scala/nl/knaw/dans/easy/fedoratobag/fixture/FileFoXmlSupport.scala#L29-L31

Reply:
Correct. I have now added a `unit test`.
How I tested it in `deasy` was first to deposit a dataset with special characters in filename and filepath, using `easy-sword2-dans-examples `module. Then I ran in deasy `easy-fedora-to-bag` using the freshly created easy-dataset:id.